docs/_build/make-javadoc.sh

```diff
@@ -19,16 +19,24 @@ export DATESTAMP=`date "+%Y-%m-%d"`
 echo "building javadoc at $DATESTAMP from:
 $SOURCE_PATHS"
 
+# BROOKLYN_VERSION_BELOW
+deps_jar=../../usage/all/target/brooklyn-all-0.7.0-SNAPSHOT-with-dependencies.jar
+if [ \! -f ${deps_jar} ]; then
+  echo "Expected to find ${deps_jar}"
+  echo "Please run a full Maven build in the project root"
+  exit 1
+fi
+
 javadoc -sourcepath $SOURCE_PATHS \
   -public \
   -d target/apidocs/ \
   -subpackages "org.apache.brooklyn:io.brooklyn:brooklyn" \
-  -classpath ../../usage/all/target/brooklyn-all-0.7.0-SNAPSHOT-with-dependencies.jar \
+  -classpath "${deps_jar}" \
   -doctitle "Apache Brooklyn" \
   -windowtitle "Apache Brooklyn" \
   -header "Apache Brooklyn" \
   -footer '<b>Apache Brooklyn - Multi-Cloud Application Management</b> <br/> <a href="http://brooklyn.io/" target="_top">brooklyn.io</a>. Apache License. &copy; '$DATESTAMP'.' \
- | tee target/javadoc.log
+2>&1 1>/dev/null | tee target/javadoc.log
```

Comment:
Can you also make sure `target` exists? If it doesn't the output isn't redirected as expected and the subsequent `grep` isn't grepping anything.